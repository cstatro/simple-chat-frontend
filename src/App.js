import React from "react";
import FunBox from "./components/FunBox";

const App = () => {
  return <div></div>;
};

export default App;
