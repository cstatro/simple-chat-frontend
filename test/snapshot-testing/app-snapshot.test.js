import React from "react";
import App from "../../src/App";
import renderer from "react-test-renderer";

test("App tree renders as expected", () => {
  const component = renderer.create(<App />);

  let tree = component.toJSON();

  expect(tree).toMatchSnapshot();

  expect(tree).toMatchSnapshot();
});
