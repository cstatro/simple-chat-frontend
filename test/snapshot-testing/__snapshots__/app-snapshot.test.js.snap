// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App tree renders as expected 1`] = `<div />`;

exports[`App tree renders as expected 2`] = `<div />`;
